src/main/java/bouncybob/util/Ui.java

```diff
@@ -0,0 +1,139 @@
+package bouncybob.util;
+
+import bouncybob.BouncyBob;
+import bouncybob.task.Task;
+
+/**
+ * Handles the user interface of the BouncyBob application.
+ */
+public class Ui {
+    private static final String TOP_BORDER = "================================================";
+    private static final String MIDDLE_BORDER = "|                                              |";
+    private static final String BOTTOM_BORDER = "================================================";
```

Comment:
Great work avoiding a common Java coding standard violation of not following the naming convention for Java constants! Much appreciated!